flexmeasures/ui/templates/base.html

```diff
@@ -649,7 +658,9 @@
 
 
     {#- Scripts used by all views (e.g. by navigation menu) -#}
-    <script src="https://ajax.aspnetcdn.com/ajax/bootstrap/3.3.7/bootstrap.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-YvpcrYf0tY3lHB60NNkmXc5s9fDVZLESaAA55NDzOxhy9GkcIdslK1eN7N6jIeHz" crossorigin="anonymous"></script>
+    <script src="https://cdn.datatables.net/2.0.8/js/dataTables.min.js"></script>
+    <script src="https://cdn.datatables.net/2.0.8/js/dataTables.bootstrap5.min.js"></script>
```

Comment:
Further up, we're downloading `https://cdn.datatables.net/buttons/3.0.2/css/buttons.bootstrap5.min.css` and `https://cdn.datatables.net/1.10.22/css/jquery.dataTables.min.css`.

Also this image: `https://cdn.datatables.net/1.10.0/images/sort_asc.png`

Do you recommend we sort out the dataTables versioning? (could also happen in another PR)
Or is this just okay as it is?

Reply:
Given the current setup, I don't recommend sorting out the dataTables versioning at this point. While it's generally a good practice to ensure consistency in versioning, the versions being used here are functional and compatible. Therefore, unless we encounter specific issues directly related to version mismatches, it might be best to leave them as they are to avoid introducing unnecessary complexity or potential disruptions. This can always be revisited in a future PR if needed.